### PR TITLE
Fix #3200 : Fixed "Get Auth Token" Button CSS

### LINKED ISSUE #3200: [FrontendV2] Buttons are not easily clickable in Get Auth Token at /profile
**Cancel**, **Download JSON** and **Refresh Token** buttons are not easily clickable in **Get Your Auth Token** Modal at `/profile`.

In order to click the above mentioned buttons, one needs to click precisely right below the button instead of clicking on them.
This problem becomes adverse in mobile screens.

If you'd like to report a bug, please include the following information:
#### Expected behaviour
Buttons should be easily clickable.

#### Observed behaviour
The button is not clickable even when you hover on them.
image

### Steps to reproduce the problem
1. Go to `/profile` on the frontend.
2. Click **Get your Auth Token**.
3. Hover over **Cancel**, **Download JSON**, or **Refresh Token** button.

**I would like to work on this issue. Thanks**

### PR DESCRIPTION
**Pull request by ShauryaAg**
Fixes #3200

Fixed CSS for **Cancel**, **Download JSON** and **Refresh Token** button in *"Get Your Auth Token"* Modal present at `/profile`

### CODE REVIEW

**Comment by ShauryaAg** [comment]
@RishabhJain2018 I have made the necessary changes for #3200, please review.

**Comment by Ram81** [comment]
@ShauryaAg can you post a GIF of how it looks after the fix?

**Comment by ShauryaAg** [comment]
Yes, absolutely.
This is what it looks after making the required changes.
localhost_4200_profile

File: frontend_v2/src/app/components/profile/profile.component.html [hunk 1]
```diff
@@ -133,7 +133,7 @@
                     <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
                   </button>
                 </div>
-                <ul class="inline-list">
+                <ul id="auth_token_inline_list" class="inline-list">
```
Comment (Ram81) [inline frontend_v2/src/app/components/profile/profile.component.html:136]: @ShauryaAg it would be good to make these as css classes instead

File: frontend_v2/src/app/components/profile/profile.component.html [hunk 2]
```diff
@@ -114,7 +114,7 @@
                   API Authentication Token
                 </div>
                 <div class="form row">
-                  <div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
+                  <div id="token_container" class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
```
Comment (Ram81) [inline frontend_v2/src/app/components/profile/profile.component.html:117]: @ShauryaAg  it would be good to make these as css classes instead

File: frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ import { GlobalService } from '../../../services/global.service';
 import { ChallengeService } from '../../../services/challenge.service';
 import { EndpointsService } from '../../../services/endpoints.service';
 import { SelectphaseComponent } from '../../utility/selectphase/selectphase.component';
-import { environment } from '../../../../environments/environment.staging';
+import { environment } from '../../../../environments/environment';
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts:14]: @ShauryaAg why is this change needed?

File: frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts [hunk 4]
```diff
@@ -91,7 +92,8 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     private globalService: GlobalService,
     private apiService: ApiService,
     private endpointsService: EndpointsService,
-    private router: Router
+    private router: Router,
+    private logger: NGXLogger
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts:96]: @ShauryaAg why are we adding a logger if we are not using it?

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts (hunk 3 above)
Comment (ShauryaAg) [reply frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts:14]: I am extremely sorry about that. I accidentally created a new branch from my previous branch for another PR. Hence, the commits for my last two PR's are reflected in this one too.

File: frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts (hunk 4 above)
Comment (ShauryaAg) [reply frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts:96]: This is also the result of my previous PR. Extremely sorry about this.

File: frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts (hunk 3 above)
Comment (ShauryaAg) [reply frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts:14]: I will revert the unnecessary changes in the next commit.

**Review by Ram81 — CHANGES_REQUESTED** [review]
@ShauryaAg can you post a screenshot of the UI after changes

**Comment by ShauryaAg** [comment]
Sure. Here's a GIF after the most recent changes.

localhost_4200_profile - Google Chrome 2021-02-08 18-23-03

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Review by RishabhJain2018 — APPROVED** [review]
